reana_commons/mail.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2020 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+"""REANA-Commons email util."""
+
+import logging
+import os
+import smtplib
+import ssl
+
+# Email configuration
+REANA_SMTP_SERVER = os.getenv('REANA_SMTP_SERVER')
+REANA_SMTP_PORT = os.getenv('REANA_SMTP_PORT')
+REANA_EMAIL_LOGIN = os.getenv('REANA_EMAIL_LOGIN')
+REANA_EMAIL_SENDER = os.getenv('REANA_EMAIL_SENDER')
+REANA_EMAIL_PASSWORD = os.getenv('REANA_EMAIL_PASSWORD')
```

Comment:
I see `REANA_SMTP_*` and `REANA_EMAIL_*` being used. What about using the same prefix for all variables? It would be easier to identify which variable belongs to which REANA module or utility across the code base in that way. For example, `REANA_DB_*`, `REANA_SMTP_*`, etc.

One option is to use SMTP for everything:

```
REANA_SMTP_SERVER
REANA_SMTP_PORT
REANA_SMTP_LOGIN
REANA_SMTP_SENDER
REANA_SMTP_PASSWORD
```

or using MAIL or EMAIL prefix:

```
REANA_MAIL_SMTPHOST
REANA_MAIL_SMPTPORT
REANA_MAIL_LOGIN
```

(We could also harmonise the Python module file name accordingly, for example `reana_commons.mailutils` or `reana_commons.emailutils` vs `reana_commons.smtputils` depending on the chosen prefix.)